terraform/main.tf

```diff
@@ -52,6 +63,39 @@ resource "aws_s3_bucket_public_access_block" "storage_bucket_public_access" {
   restrict_public_buckets = false
 }
 
+resource "aws_s3_bucket" "storage_bucket_replication" {
+  provider = aws.backup
+  bucket   = "${local.prefix}-storagebucket-replication-${local.short_uuid}"
+  acl      = "private"
+
+  versioning {
+    enabled = true
+  }
+
+  lifecycle_rule {
+    enabled = true
+
+    transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    transition {
+      days          = 60
+      storage_class = "GLACIER"
+    }
+  }
```

Comment:
This generally looks good - it is very annoying that the glacier minimum capacity charge per object is 40KB. Most of our objects are way, way less. This is going to dramatically inflate costs. 

I think we should use standard and then auto transition to standard infrequent access after 30 days, but not transition to glacier for now. Maybe this makes sense later, but I think for now it will cost more than using standard infrequent access because of the minimum capacity charge.

Reply:
I was actually thinking the same thing, but really only because it's much easier/quicker to restore them to Standard Access.